### PR TITLE
Add Finbuckle.MultiTenant

### LINKED ISSUE #30472: additional multi-tenant authentication option request
Hello,
I maintain an established .NET Foundation open source project for multi-tenancy: [Finbuckle.MultiTenant](https://www.finbuckle.com/multitenant)

Would it be possible to get it mentioned in the documentation alongside ABP and Orchard with respect to multi-tenant authentication support? Something like:

[Finbuckle.MultiTenant](https://www.finbuckle.com/multitenant) is a lightweight .NET multi-tenancy library which supports per-tenant authentication configuration.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 393b3379-6061-50a4-0975-b3d1d82b660b
* Version Independent ID: 48e4e204-ea22-c225-0cbc-0d811bdb90ed
* Content: [Overview of ASP.NET Core Authentication](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/?view=aspnetcore-7.0#defaultscheme)
* Content Source: [aspnetcore/security/authentication/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @mjrousos
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #30472

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/authentication/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/326c4b3109c5837a3d83576799082f30ef99f24d/aspnetcore/security/authentication/index.md) | [Overview of ASP.NET Core authentication](https://review.learn.microsoft.com/en-us/aspnet/core/security/authentication/index?branch=pr-en-us-30486) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@AndrewTriesToCode use the suggest changes feature to make suggestions.

File: aspnetcore/security/authentication/index.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ See the following links for differences between challenge and forbid:
 
 ## Authentication providers per tenant
 
-ASP.NET Core doesn't have a built-in solution for multi-tenant authentication. While it's possible for customers to write one using the built-in features, we recommend customers to consider [Orchard Core](https://www.orchardcore.net/) or [ABP Framework](https://abp.io/) for multi-tenant authentication.
+ASP.NET Core doesn't have a built-in solution for multi-tenant authentication. While it's possible for customers to write one using the built-in features, we recommend customers to consider [Orchard Core](https://www.orchardcore.net/), [ABP Framework](https://abp.io/), or [Finbuckle.MultiTenant](https://www.finbuckle.com/multitenant)  for multi-tenant authentication.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/index.md:140]:
```suggestion
ASP.NET Core doesn't have a built-in solution for multi-tenant authentication. While it's possible for customers to write one using the built-in features, we recommend customers consider [Orchard Core](https://www.orchardcore.net/), [ABP Framework](https://abp.io/), or [Finbuckle.MultiTenant](https://www.finbuckle.com/multitenant)  for multi-tenant authentication.
```

File: aspnetcore/security/authentication/index.md [hunk 2]
```diff
@@ -148,6 +148,15 @@ See the [Orchard Core](https://github.com/OrchardCMS/OrchardCore) source for an
 
 [ABP Framework](https://abp.io/) supports various architectural patterns including modularity, microservices, domain driven design, and multi-tenancy. See [ABP Framework source on GitHub](https://github.com/abpframework/abp).
 
+Finbuckle.MultiTenant:
+
+* Open source
+* Provides tenant resolution
+* Lightweight
+* Provides data isolation
+* Configure app behavior uniquely for each tenant
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/index.md:157]: @AndrewTriesToCode  I just took those from the web page. You might want to have a list better differentials Finbuckle.MultiTenant from the previous alternatives.

**Comment by mkArtakMSFT** [comment]
@sebastienros  can you please review this? Thanks!

**Review by sebastienros — APPROVED** [review]

**Pull request merged**